pkg/controller/bookkeepercluster/bookie.go

```diff
@@ -203,35 +203,64 @@ func makeBookiePodSpec(bk *v1alpha1.BookkeeperCluster) *corev1.PodSpec {
 	return podSpec
 }
 
-func createVolumeMount(ledgerDirs []string, journalDirs []string, indexDirs []string) []corev1.VolumeMount {
+func createVolumeMount(bk *v1alpha1.BookkeeperCluster, ledgerDirs []string, journalDirs []string, indexDirs []string) []corev1.VolumeMount {
```

Comment:
we can remove passing bookkeeper cluster from the function prototype

Reply:
done.